core/engine/src/vm/code_block.rs

```diff
@@ -69,10 +69,19 @@ bitflags! {
 
         /// Arrow and method functions don't have `"prototype"` property.
         const HAS_PROTOTYPE_PROPERTY = 0b1000_0000;
+    }
+}
 
+#[cfg(feature = "trace")]
+bitflags! {
+    /// Trace specific flags for [`CodeBlock`]
+    #[derive(Clone, Copy, Debug, Finalize)]
+    pub(crate) struct TraceFlags: u8 {
         /// Trace instruction execution to `stdout`.
-        #[cfg(feature = "trace")]
-        const TRACEABLE = 0b1000_0000_0000_0000;
+        const TRACEABLE = 0b0000_0001;
+
+        /// Has the `CodeBlock` been traced.
+        const CALLFRAME_TRACED = 0b0000_0010;
     }
 }
```

Comment:
Yeah, I can switch it over. I was sort of leaving at as a separate `u8`, so that we could add more options in the future and not constrain any flags added to `CodeBlockFlags`. That may have been premature though.